editor/modes/visual-editor/block.js

```diff
@@ -229,6 +232,7 @@ export default connect(
 			block: getBlock( state, ownProps.uid ),
 			isSelected: isBlockSelected( state, ownProps.uid ),
 			isHovered: isBlockHovered( state, ownProps.uid ),
+			isInsertionPoint: isBlockBeforeInsertPoint( state, ownProps.uid ),
```

Comment:
I believe we should create a new element at the block insertion point, otherwise the insertion point will be determined by the specificities of the block-before-insertion-point. Example: if the block is a wide image, the blue line will be full width.